package.json

```diff
@@ -27,71 +27,65 @@
+    "aws-sdk-client-mock": "^4.0.0",
+    "esbuild": "^0.20.2",
+    "nock": "^13.5.4",
+    "snyk": "^1.1283.0",
+    "vitest": "^1.4.0"
   },
   "dependencies": {
-    "@apollo/server": "^4.4.1",
-    "@as-integrations/aws-lambda": "^2.0.1",
-    "@aws-sdk/client-lambda": "^3.405.0",
-    "@aws-sdk/client-s3": "^3.282.0",
-    "@babel/core": "^7.21.0",
-    "@babel/eslint-parser": "^7.22.15",
-    "@babel/preset-env": "^7.20.2",
+    "@apollo/server": "^4.10.2",
+    "@as-integrations/aws-lambda": "^3.1.0",
+    "@aws-sdk/client-lambda": "^3.540.0",
+    "@aws-sdk/client-s3": "^3.540.0",
+    "@babel/core": "^7.24.3",
+    "@babel/eslint-parser": "^7.24.1",
+    "@babel/preset-env": "^7.24.3",
     "@edsc/eslint-config": "^0.0.5",
-    "@graphql-tools/merge": "^8.2.10",
-    "@graphql-tools/schema": "^8.5.1",
+    "@graphql-tools/merge": "^9.0.3",
+    "@graphql-tools/schema": "^10.0.3",
     "array-foreach-async": "^1.0.1",
-    "axios": "^1.6.4",
-    "babel-loader": "^8.3.0",
-    "camelcase-keys": "^7.0.2",
-    "clean-webpack-plugin": "^4.0.0",
+    "axios": "^1.6.8",
+    "babel-loader": "^9.1.3",
+    "camelcase-keys": "^9.1.3",
     "dasherize": "^2.0.0",
     "dataloader": "^2.2.2",
-    "eslint": "^8.35.0",
+    "eslint": "^8.57.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-plugin-cypress": "^2.14.0",
-    "eslint-plugin-import": "^2.28.1",
-    "eslint-plugin-import-newlines": "^1.3.4",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-react": "^7.33.2",
-    "eslint-webpack-plugin": "^3.2.0",
-    "graphql": "^16.6.0",
+    "eslint-plugin-cypress": "^2.15.1",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-import-newlines": "^1.4.0",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-react": "^7.34.1",
+    "graphql": "^16.8.1",
     "graphql-parse-resolve-info": "^4.13.0",
     "graphql-tag": "^2.12.6",
     "graphql-type-json": "^0.3.2",
-    "jsonwebtoken": "^9.0.0",
-    "jwks-rsa": "^2.1.5",
+    "jsonwebtoken": "^9.0.2",
+    "jwks-rsa": "^3.1.0",
     "lodash": "^4.17.21",
-    "qs": "^6.11.0",
-    "serverless": "^3.37.0",
-    "serverless-offline": "^12.0.4",
-    "serverless-plugin-log-subscription": "^2.2.0",
+    "qs": "^6.12.0",
+    "serverless": "^3.38.0",
+    "serverless-esbuild": "^1.52.1",
+    "serverless-offline": "^13.3.3",
+    "serverless-plugin-log-subscription": "^2.2.1",
     "serverless-python-requirements": "github:william-valencia/serverless-python-requirements#master",
-    "serverless-webpack": "^5.11.0",
-    "snakecase-keys": "^5.4.5",
-    "stellate": "^2.5.2",
-    "uuid": "^9.0.0",
-    "webpack": "^5.76.1",
-    "webpack-node-externals": "^3.0.0",
-    "ws": "^8.12.1"
+    "snakecase-keys": "^6.0.0",
+    "stellate": "^2.11.1",
+    "uuid": "^9.0.1",
+    "ws": "^8.16.0"
   },
   "overrides": {
     "graphql-parse-resolve-info": {
-      "graphql": "^16"
+      "graphql": "$graphql"
```

Comment:
What does this do? It's a vastly different value than it was previously

Reply:
That is how we do the overrides in other apps, it tells npm to use the version of `graphql` we specified when we installed the package. So if we upgrade it at some point using `$graphql` keeps us from having to change this value to `^17`

Reply:
Ah, neat.